test/parallel/test-repl-require-context.js

```diff
@@ -13,7 +13,7 @@ child.stdout.on('data', (data) => {
 });
 
 child.on('exit', common.mustCall(() => {
-  const results = output.replace(/^> /mg, '').split('\n');
+  const results = output.replace(/^> /mg, '').split('\n').filter((v, i) => i > 1);
   assert.deepStrictEqual(results, ['undefined', 'true', 'true', '']);
```

Comment:
Instead of filtering the output, I suggest to actually check for the welcome message here as well.